ObjectiveGit/GTIndex.m

```diff
@@ -308,17 +316,17 @@ - (BOOL)enumerateConflictedFilesWithError:(NSError **)error usingBlock:(void (^)
 			return NO;
 		}
 
-		GTIndexEntry *blockAncestor;
+		GTIndexEntry *blockAncestor = nil;
 		if (ancestor != NULL) {
 			blockAncestor = [[GTIndexEntry alloc] initWithGitIndexEntry:ancestor];
 		}
 
-		GTIndexEntry *blockOurs;
+		GTIndexEntry *blockOurs = nil;
```

Comment:
While I understand the compiler makes it easier by auto-nulling new declarations, I don't think an implicit `= nil` makes any harm. :)